azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/DependencyInjectionContainerHolder.kt

```diff
@@ -75,8 +75,7 @@ internal class DependencyInjectionContainerHolder(
                 ParticipantGridCellViewModelFactory(),
                 application.resources.getInteger(R.integer.azure_communication_ui_calling_max_remote_participants),
                 container.debugInfoManager,
-                container.configuration.enableMultitasking,
-                container.configuration.telecomOptions != null
+                container.configuration
```

Comment:
I think `container.configuration.telecomOptions != null` is removed while solving merge conflict, we can add back

Reply:
azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/factories/CallingViewModelFactory.kt

It's still there, just moved.

Since container.configuration is needed for a bunch of VM, I just pass the entire config through now.